### DISCUSSION TITLE
Can I use Angular framework?

### DISCUSSION THREAD

**Discussion by reuse-ay** [wxt-dev/wxt#818] [Q&A]
As an angular developer, I am wondering if Angular can be used with wxt. Vite is available in recent versions of Angular.

**Comment by aklinker1** [comment]
As long as there's a vite plugin you can add to the `wxt.config.ts` file, it should be fine. But I don't think that's how angular works...

You don't have to use the WXT modules for the frontend, you can just use vite plugins. The WXT modules just add some default config.

For example, react, is as simple as this:

```ts
// wxt.config.ts
import { defineConfig } from 'wxt';
import react from '@vitejs/plugin-react';

export default defineConfig({
  vite: () => ({
    plugins: [react()],
  }),
});
```

If a similar plugin for angular exists, you may be able to do something similar.

**Reply by reuse-ay** [reply]
Thank you! Analogjs seems to have [vite plugin for Angular](https://www.npmjs.com/package/@analogjs/vite-plugin-angular). I'm going to give it a try.

**Reply by felipedrumond** [reply]
@reuse-ay how did it go? Can you share how you managed to get angular app built when running _npm run dev_?
